### PR TITLE
Add quantity field to the food model

### LINKED ISSUE #88: As a user, I want to update when I use up ingredients so that I have an updated record of the food available in my kitchen

### PR DESCRIPTION
**Pull request by sagarsureka**

### CODE REVIEW

File: src/main/java/seedu/simplykitchen/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -21,11 +22,13 @@
             + PREFIX_DESCRIPTION + "DESCRIPTION "
             + PREFIX_PRIORITY + "PRIORITY "
             + PREFIX_EXPIRY_DATE + "EXPIRY DATE "
+            + PREFIX_QUANTITY + "QUANTITY "
             + "[" + PREFIX_TAG + "TAG]...\n  "
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_DESCRIPTION + "Canned Tuna "
             + PREFIX_PRIORITY + "HIGH "
             + PREFIX_EXPIRY_DATE + "1-1-2020 "
+            + PREFIX_QUANTITY + "100 gm "
```
Comment (songyiang) [inline src/main/java/seedu/simplykitchen/logic/commands/AddCommand.java]: A very small nitpick but "g" will be a better unit. Also, perhaps consider removing the "20g" tag in line 33?

File: src/main/java/seedu/simplykitchen/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -90,8 +93,9 @@ private static Food createEditedFood(Food foodToEdit, EditFoodDescriptor editFoo
         Description updatedDescription = editFoodDescriptor.getDescription().orElse(foodToEdit.getDescription());
         Priority updatedPriority = editFoodDescriptor.getPriority().orElse(foodToEdit.getPriority());
         ExpiryDate updatedExpiryDate = editFoodDescriptor.getExpiryDate().orElse(foodToEdit.getExpiryDate());
+        Quantity updateQuantity = editFoodDescriptor.getQuantity().orElse(foodToEdit.getQuantity());
         Set<Tag> updatedTags = editFoodDescriptor.getTags().orElse(foodToEdit.getTags());
-        return new Food(updatedDescription, updatedPriority, updatedExpiryDate, updatedTags);
+        return new Food(updatedDescription, updatedPriority, updatedExpiryDate, updateQuantity, updatedTags);
```
Comment (songyiang) [inline src/main/java/seedu/simplykitchen/logic/commands/EditCommand.java]: Using "updatedQuantity" will make it consistent with the naming of the other variables.

File: src/main/java/seedu/simplykitchen/model/util/SampleDataUtil.java [hunk 3]
```diff
@@ -19,17 +20,17 @@
...
+                    new Quantity("1 pie"), getTagSet("Frozen")),
             new Food(new Description("Bread"), new Priority("medium"), new ExpiryDate("1-10-2020"),
-                    getTagSet("Wholemeal", "400g")),
+                    new Quantity("2 loaf"), getTagSet("Wholemeal", "400g")),
             new Food(new Description("Chocolate Cake"), new Priority("high"), new ExpiryDate("1-1-2021"),
-                    getTagSet("$50")),
+                    new Quantity("0.7 lb"), getTagSet("$50")),
             new Food(new Description("Donuts"), new Priority("low"), new ExpiryDate("1-11-2021"),
-                    getTagSet("Plain")),
+                    new Quantity("5 donuts"), getTagSet("Plain")),
```
Comment (songyiang) [inline src/main/java/seedu/simplykitchen/model/util/SampleDataUtil.java]: Is using the singular verb "donut" be better? What if the same user add another food item of 1 donut?

File: src/test/data/JsonSerializableFoodInventoryTest/typicalFoodInventory.json [hunk 4]
```diff
@@ -4,36 +4,43 @@
...
     "description": "French Fries",
     "priority": "LOW",
     "expiryDate" : "31-1-2022",
+    "quantity": "2 packet",
     "tagged" : [ ]
   }, {
     "description": "Ginger",
     "priority": "LOW",
     "expiryDate" : "31-12-2022",
+    "quantity": "50 gm",
```
Comment (songyiang) [inline src/test/data/JsonSerializableFoodInventoryTest/typicalFoodInventory.json]: Same as previous with regards to using "g" as a unit.

**Review by songyiang — COMMENTED** [review]
Just a few comments.

**Comment by sagarsureka** [comment]
Thank you for your comments. I have made the relevant changes to the code.

**Pull request merged**